### PR TITLE
Link profile link in header to profile component

### PR DESCRIPTION
**Pull request by TheunisK**

### CODE REVIEW

File: src/client/components/Profile.jsx
Comment (vherus) [inline src/client/components/Profile.jsx:23]: This should probably initialise to an empty array or you'll encounter problems with `posts.map` trying to run .map on `null` during initial render

**Review by vherus — APPROVED** [review]
Highly advise you update this PR with the suggestion I've posted but no need for a second review once you do so

File: src/client/components/Profile.jsx
Comment (TheunisK) [reply src/client/components/Profile.jsx:23]: Noted. Thanks Nathan.

**Pull request merged**
